src/languages/en.ts

```diff
@@ -2201,7 +2201,6 @@ const translations = {
         whatsYourLegalName: 'What’s your legal name?',
         whatsYourDOB: 'What’s your date of birth?',
         whatsYourAddress: 'What’s your address?',
-        noPOBoxesPlease: 'No PO boxes or mail-drop addresses, please.',
```

Comment:
Sam here, this will need to be updated in es.ts

Reply:
Looking good?